### DISCUSSION TITLE
Allow building certain walls underground with clearance checks disabled

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16606] [Feature requests] [answered]
When you have clearance checks disabled, walls that can be placed on a slant and some other walls still check whether you are above ground. The waterfall is the most common type of wall that  checks this. If you removed this check when the cheat was enabled, then there would be less need to use the tile inspector to lower these particular walls underground

**Comment by J-PIE-314** [comment]
Waterfalls can now be lowered and built underground using the shift key while "Disable clearance checks" is active.

Screenshot example (invisible base land):
![image of waterfalls underground](https://github.com/user-attachments/assets/50630610-3c0e-40d8-bb28-bfb3cb0cda0c)

**Comment by 733737** [comment, marked as answer]
this was fixed around a year ago, I forgot to update this discussion
